packages/link/src/Link.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+
+type Props = {
+  /** Content of the link. Can be anything */
+  children: MnrlReactNode,
+  /** URI tied to the Link */
+  to: string
+};
+
+function transform(measurement: string, func: (number) => number) {
+  const value = parseFloat(measurement);
+  const units = measurement.replace(value, '');
+  return `${func(value)}${units}`;
+}
+
+const Root = createStyledComponent(RouterLink,
+  (props, theme) => {
+    return {
+      color: theme.color_link,
+      fontWeight: theme.Link_fontWeight || theme.fontWeight_regular,
+      fontSize: theme.Link_fontSize || transform(theme.fontSize_base, v => v*2),
+
+      '&:hover': {
+        color: theme.color_link_hover
+      },
+      '&:focus': {
+        color: theme.color_link_focus
+      },
+      // `:active` must be last, to follow LVHFA order:
+      // https://developer.mozilla.org/en-US/docs/Web/CSS/:active
+      '&:active': {
+        color: theme.color_link_active
+      }
+    };
+  },
+  {
+    displayName: 'Link',
+    includeStyleReset: true
+  }
+);
+
+/**
+ * Link component
```

Comment:
Perhaps we can find a better way to describe this component, rather than simply Link component.  Check here for some inspiration.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a